RELEASE/scripts/autoscend/auto_util.ash

```diff
@@ -1247,7 +1247,7 @@ boolean adjustForBanish(string combat_string)
 
 boolean adjustForBanishIfPossible(monster enemy, location loc)
 {
-	if(canBanish(enemy, loc))
+	if(canBanish(enemy, loc) && !in_pokefam()) // No banishes in Pocket Familiars, so removing this from options.
```

Comment:
just make canBanish return false when in pokefam

Reply:
Attempted to fix this via most recent commit